src/test/java/seedu/address/util/NetworkTest.java

```diff
@@ -84,13 +83,73 @@ public void fetchAsString_success() {
     @Test
     public void fetchAsString_invalidUrl_throwsIoexception() throws IOException {
         thrown.expect(IOException.class);
-        fetchAsString(TestUtil.toUrl("https://abc.``ILLEGAL_CHARS.com"));
+        Network.fetchAsString(TestUtil.toUrl("https://abc.``ILLEGAL_CHARS.com"));
     }
 
     @Test
     public void fetchAsString_invalidWebsite_throwsIoexception() throws IOException {
         thrown.expect(IOException.class);
-        fetchAsString(TestUtil.toUrl("https://thiswebsite.does.not.exist.definitely"));
+        Network.fetchAsString(TestUtil.toUrl("https://thiswebsite.does.not.exist.definitely"));
+    }
+
+    @Test
+    public void fetchArticle_success() throws IOException {
+        String httpsContent = new String(Network.fetchArticle(
+            TestUtil.toUrl("https://cs2103-ay1819s2-w10-1.github.io/main/networktests/")));
+        assertTrue(httpsContent.length() > 0);
+        assertTrue(httpsContent.contains("<p>It works!</p>"));
+
+        String httpContent = new String(Network.fetchArticle(
+            TestUtil.toUrl("http://cs2103-ay1819s2-w10-1.github.io/main/networktests/")));
+        assertTrue(httpContent.length() > 0);
+
+        String localContent = new String(Network.fetchArticle(
+            MainApp.class.getResource("/NetworkTest/default.html")));
+        assertTrue(localContent.length() > 0);
+    }
+
+    @Test
+    public void fetchAsBytesAsync_success() throws ExecutionException, InterruptedException {
+        Network.fetchAsBytesAsync(
+            TestUtil.toUrl("https://cs2103-ay1819s2-w10-1.github.io/main/networktests/"))
+            .thenAccept(content -> {
+                String httpsContent = new String(content);
+                assertTrue(httpsContent.length() > 0);
+                assertTrue(httpsContent.contains("<p>It works!</p>"));
+            })
+            .get();
+
+        Network.fetchAsBytesAsync(
```

Comment:
duplicate test?

Reply:
The one above is https while this one is http.